src/PointwiseFunctions/GeneralRelativity/Christoffel.hpp

```diff
@@ -107,6 +107,27 @@ struct TraceSpatialChristoffelFirstKindCompute
   using base = TraceSpatialChristoffelFirstKind<SpatialDim, Frame, DataType>;
 };
 
+/// Compute item for the trace of the spatial Christoffel symbols
+/// of the second kind
+/// \f$\Gamma^{i} = \Gamma^{i}_{jk}g^{jk}\f$ computed from the
+/// Christoffel symbols of the second kind and the inverse spatial metric.
+///
+/// Can be retrieved using `gr::Tags::TraceSpatialChristoffelSecondKind`
+template <size_t SpatialDim, typename Frame, typename DataType>
+struct TraceSpatialChristoffelSecondKindCompute
+    : TraceSpatialChristoffelSecondKind<SpatialDim, Frame, DataType>,
+      db::ComputeTag {
+  using argument_tags =
+      tmpl::list<SpatialChristoffelSecondKind<SpatialDim, Frame, DataType>,
+                 InverseSpatialMetric<SpatialDim, Frame, DataType>>;
+  static constexpr tnsr::I<DataType, SpatialDim, Frame> (*function)(
+      const tnsr::Ijj<DataType, SpatialDim, Frame>&,
+      const tnsr::II<DataType, SpatialDim, Frame>&) =
+      &trace_last_indices<DataType, SpatialIndex<SpatialDim, UpLo::Up, Frame>,
+                          SpatialIndex<SpatialDim, UpLo::Lo, Frame>>;
+  using base = TraceSpatialChristoffelSecondKind<SpatialDim, Frame, DataType>;
```

Comment:
Please remind me what `using base =` is for?

Reply:
`base` is used to know the `SimpleTag` that any given `ComputeTag` is derived off of, since c++ doesn't allow for determination of base class(es) from derived.